tests/test_diffraction_objects.py

```diff
@@ -241,40 +236,6 @@ def test_dump(tmp_path, mocker):
 
 
 tc_params = [
-    (
-        {},
```

Comment:
Where are these things now getting tested?

Reply:
pls see `test_init_invalid_args` below

Reply:
@sbillinge I think these aren't valid since we `DiffractionObject` requires x, y, and xtype (current PR)